src/minio/core/policy_creator.py

```diff
@@ -26,8 +26,9 @@
 
 3. GROUP POLICIES (PolicyType.GROUP_HOME):
    - Policy Name: <GROUP_POLICY_PREFIX><groupname>
-   - Grants WRITE access to group's shared workspace:
-     * s3a://{bucket}/groups-general-warehouse/{groupname}/
+   - Grants WRITE access to group's shared workspaces:
+     * s3a://{bucket}/tenant-sql-warehouse/{groupname}/ (for Spark tables)
+     * s3a://{bucket}/tenant-general-warehouse/{groupname}/ (for general files)
```

Comment:
UUIC the plan is for multiple tenants to share the minio instance so in future this might need changes

Reply:
Actually I assume groupname here is the tenant name, so never mind